File: src/leetcode/p2011/java/Solution.java
package leetcode.p2011.java;

public class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int answer = 0;
        for (int i = 0; i < operations.length; i++) {
            if (operations[i].charAt(0) == '+' || operations[i].charAt(2) == '+') {
                answer++;
            } else {
                answer--;
            }
        }
        return answer;
    }
}

File: src/leetcode/p2011/kotlin/Solution.kt
package leetcode.p2011.kotlin

class Solution {
    fun finalValueAfterOperations(operations: Array<String>): Int {
        var answer: Int = 0
        for (operation in operations) {
            if (operation == "X++" || operation == "++X") {
                answer++
            } else {
                answer--
            }
        }
        return answer;
    }
}